src/sr/discord_bot/bot.py

```diff
@@ -98,40 +98,64 @@ async def setup_hook(self) -> None:
 
     async def on_ready(self) -> None:
         self.logger.info(f"{self.user} has connected to Discord!")
-        guild = self.get_guild(self.guild.id)
-        if guild is None:
-            logging.error(f"Guild {self.guild.id} not found!")
+
+        if (guild := self.get_guild(self.guild.id)) is None:
+            self.logger.error("Guild not found: %r", self.guild.id)
```

Comment:
suggestion: should this be `logger.fatal` if we're going to `exit` immediately afterwards? (also for the similar case below)